src/generator/predefined/component.js

```diff
@@ -0,0 +1,66 @@
+const createTemplate = module.parent.createTemplate;
+
+const classComponentTemplate = createTemplate`
+/* @flow */
+import React, { Component } from "react";
+
+export default class ${args => args.name} extends Component {
+  render () {
+    return (
+      <div>
+        ${args => args.name}
+      </div>
+    );
+  }
+}
+`;
+
+const functionalComponentTemplate = createTemplate`
+/* @flow */
+import React from "react";
+
+export default function ${args => args.name} () {
+  return (
+    <div>
+       ${args => args.name}
+    </div>
+  );
+}
+`;
+
+const testTemplate = createTemplate`
+/*global React*/
+/*global describe it*/
+/*global expect*/
+import ${args => args.name} from "${args => args.path}";
+import { shallow } from "enzyme";
+
+describe("${args => args.name}", () => {
```

Comment:
This should be the path `components/Todos` rather than `Todos`… I realize our previous template did not include `src/` in front but I would be okay if it does now since generators can generate files all over the place.